pkg/web/httpservers.go

```diff
@@ -88,11 +89,11 @@ func NewHttpServer(
 	}
 
 	if enableProf {
-		profiler := &traceProfiler{}
 		routes = append(routes,
-			route{path: "/memprof", handler: profiler.MemProf, method: "POST", name: "profmem_post"},
-			route{path: "/pprof", handler: profiler.PProf, method: "POST", name: "profpprof_post"},
-			route{path: "/trace", handler: profiler.Trace, method: "POST", name: "proftrace_post"},
+			route{path: "/debug/pprof/", handler: pprof.Index, method: "GET", name: "pprof_index"},
+			route{path: "/debug/pprof/cmdline", handler: pprof.Cmdline, method: "GET", name: "pprof_cmdline"},
+			route{path: "/debug/pprof/profile", handler: pprof.Profile, method: "GET", name: "pprof_profile"},
+			route{path: "/debug/pprof/symbol", handler: pprof.Symbol, method: "GET", name: "pprof_symbol"},
```

Comment:
- [ ] also needs to support POST.

`pprof.Symbol` has explicit handling for POST and non-POST, the other handlers all don't check what method is used.  I'm not sure what `go tool pprof` actually sends though, so it's probably fine.

I'm also not sure if the router will let you just do...
```
			route{path: "/debug/pprof/symbol", handler: pprof.Symbol, method: "POST", name: "pprof_symbol_post"},
```
... or if it will get confused by the duplicate path.  It should be easy enough to change https://github.com/atlassian/gostatsd/blob/master/pkg/web/httpservers.go#L153 to take an array with `.Methods(route.methods...)` though.

Reply:
tested this approach and both GET and POST work fine on the trace endpoint.